### DISCUSSION TITLE
Dev Version Broken 😕

### DISCUSSION THREAD

**Discussion by Baillie521** [flybywiresim/aircraft#2295] [Help] [answered]
Heya!

I know that I’m gonna get told that the Dev version is unstable but it will go from working to not working and then not working just for me when others are able to run it.

I just wanted to see why on the current dev version the batteries always read 0v and the screens will not turn on?

Btw, this is no criticism and I know I’m asking a lot, I am loving the project!!

**Comment by nistei** [comment]
Do you have any other mods installed?

**Comment by Saschl** [comment]
This sounds like a mod conflict. Try removing everything from your community folder and then reinstall the latest dev :)

**Comment by ImenesFBW** [comment, marked as answer]
That does sound like a mod conflict. Try it with only the A32NX mod in your community folder and see if the problem persists.

**Comment by Baillie521** [comment]
Thank you all guys, I’ll give that a try after my flight and get back to you all!

**Reply by ZigTag** [reply]
How did it go?

**Reply by Baillie521** [reply]
That was perfect, thank you!
